CONTRIBUTING.md

```diff
@@ -155,11 +156,10 @@ _NB! Only release a new version of a library if dependencies in `@folkehelseinst
 
 ##### Release branches for the Fhi.Frontend.Demo app
 
-Currently no need for a release branch, we just:
+There is no need for a release branch, since the branch `dev` represents the "truth". Therefore we do not create a pull request either, we just:
 
-1. Create pull request from `dev` into `main`.
-2. After approved review, **merge** to `main` (deploy).
-   >_NB! Just merge to `main`, do not squash! We want to keep the full history in `main`._
+1. Merge `dev` into `main`
```

Comment:
Should there be a routine to make sure the Demo app is pointing to the latest release of the Style library in this section?